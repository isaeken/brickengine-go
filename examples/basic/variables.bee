let name = vars.hostname | "default-host"
let id = uuid()

let output = {
  hostname: name,
  instance_id: id
}

return output
