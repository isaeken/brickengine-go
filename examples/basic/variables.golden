map[hostname:testing instance_id:UUID]
